pkgs/by-name/pl/playdate-sdk/package.nix

```diff
@@ -0,0 +1,91 @@
+{
+  autoPatchelfHook,
+  copyDesktopItems,
+  fetchurl,
+  lib,
+  makeDesktopItem,
+  makeWrapper,
+  pkgs,
+  stdenv,
+}:
+
+let
+  version = "2.5.0";
+in
+
+stdenv.mkDerivation rec {
+  pname = "playdate-sdk";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://download.panic.com/playdate_sdk/Linux/PlaydateSDK-${version}.tar.gz";
```

Comment:
I haven't seems elsewhere use this variable.

BC you have use rec, it's unnecessary to declaim this variables in let.

Reply:
```suggestion
stdenv.mkDerivation rec {
  pname = "playdate-sdk";
  version = "2.5.0";

  src = fetchurl {
    url = "https://download.panic.com/playdate_sdk/Linux/PlaydateSDK-${version}.tar.gz";
```